exercises/simple-linked-list/Hints.md

```diff
@@ -0,0 +1,4 @@
+## Hints
+This exercise requires you to process a collection of data using linked list data structure. In order to do this, you might find yourselves in need to iterate the collection.
```

Comment:
Actually, people *have* to implement `IEnumerable`, so I would rephrase this to something like: "This exercise requires you to create a linked list data structure which can be iterated. This requires you to implement the `IEnumerable<T>` interface". For more information, see [this page] (https://msdn.microsoft.com/en-us/library/9eekhta0(v=vs.110).aspx)."